pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -618,11 +618,11 @@ def fields_transform(fields):
     def _render_jira_template(fields: Dict, template_vars: Dict):
         fields.copy()
         try:
-            fields["summary"] = jinja2.Template(fields["summary"]).render(template_vars)
+            fields["summary"] = jinja2.Template(fields["summary"], autoescape=True).render(template_vars)
```

Comment:
This warning doesn't make sense to me either. `autoescape` is for escaping HTML. I don't think we can use HTML in JIRA.

Reply:
Similarly, snyk thinks the jinja2.Template method should have autoescape argument, it didn't check the context